docs/src/TapLink.doc.js

```diff
@@ -0,0 +1,132 @@
+// @flow strict
+import React from 'react';
+import PropTable from './components/PropTable.js';
+import Example from './components/Example.js';
+import PageHeader from './components/PageHeader.js';
+
+const cards = [];
+const card = c => cards.push(c);
+
+card(
+  <PageHeader
+    name="TapLink"
+    description="TapLink allows you to show links in an accessible way. It has cursor & focus styles and triggers `onTap` when hitting Space / Enter."
+  />
+);
+
+card(
+  <PropTable
+    props={[
+      {
+        name: 'children',
+        type: 'React.Node',
+      },
+      {
+        name: 'hoverStyle',
+        type: `'none' | 'underline'`,
+        defaultValue: 'none',
+      },
+      {
+        name: 'href',
+        type: 'string',
+        required: true,
+      },
+      {
+        name: 'inline',
+        type: 'boolean',
+        defaultValue: false,
+      },
+      {
+        name: 'onTap',
+        type:
+          '({ event: SyntheticMouseEvent<HTMLAnchorElement> | SyntheticKeyboardEvent<HTMLAnchorElement> }) => void',
+        href: 'basicExample',
+      },
+      {
+        name: 'rel',
+        type: `"none" | "nofollow"`,
+      },
+      {
+        name: 'rounding',
+        type: `"pill" | "circle" | 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8`,
+        href: 'basicExample',
+      },
+      {
+        name: 'target',
+        type: `"null" | "self" | "blank"`,
+        defaultValue: 'null',
+      },
+    ]}
+  />
+);
+
+card(
+  <Example
+    id="basicExample"
+    name="Example"
+    defaultCode={`
+function BasicExample() {
+  const [preventDefault, setPreventDefault] = React.useState(true);
+  const onTap = ({ event }) => {
+    if (preventDefault) {
+      event.preventDefault();
+    }
+  };
+
+  return (
+    <Box>
+      <Box display="flex" alignItems="center" padding={1} marginStart={-2} marginEnd={-2}>
+        <Box paddingX={2}>
+          <Label htmlFor="preventDefault">
+            <Text>Prevent default on tap</Text>
+          </Label>
+        </Box>
+        <Switch
+          preventDefault
```

Comment:
@jackhsu978 should this have been `id="preventDefault"` to link the label to the switch?

Reply:
good catch! done!